patches/assimp.patch

```diff
@@ -9,3 +11,31 @@
      "Build a static (.a) version of the library" )
  
  SET(ASSIMP_DEBUG_POSTFIX "d" CACHE STRING "Debug Postfitx for lib, samples and tools")
+diff --git libs/assimp-3.1.1/code/BlenderDNA.cpp libs/assimp-3.1.1/code/BlenderDNA.cpp
+index 16e639e..53e9706 100644
+--- libs/assimp-3.1.1/code/BlenderDNA.cpp
++++ libs/assimp-3.1.1/code/BlenderDNA.cpp
+@@ -55,7 +55,7 @@ using namespace Assimp::Formatter;
+ 
+ #define for_each BOOST_FOREACH
+ bool match4(StreamReaderAny& stream, const char* string) {
+-	char tmp[] = { 
++	int8_t tmp[] = { 
+ 		(stream).GetI1(), 
+ 		(stream).GetI1(),  
+ 		(stream).GetI1(), 
+@@ -344,10 +344,10 @@ void SectionParser :: Next()
+ 	stream.SetCurrentPos(current.start + current.size);
+ 
+ 	const char tmp[] = {
+-		stream.GetI1(),
+-		stream.GetI1(),
+-		stream.GetI1(),
+-		stream.GetI1()
++		(char) stream.GetI1(),
++		(char) stream.GetI1(),
++		(char) stream.GetI1(),
++		(char) stream.GetI1()

```

Comment:
Try using `static_cast` in cases like these if possible: https://en.cppreference.com/w/cpp/language/static_cast.

This would be `static_cast<char>(stream.GetI1()),`.